common/pom.xml

```diff
@@ -8,8 +8,22 @@
         <version>1.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
-
     <artifactId>common</artifactId>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot</artifactId>
+            <version>1.5.7.RELEASE</version>
```

Comment:
It'd be nice if you use dependency management, and also please extract the version values into pom properties.

Reply:
Ok thanks